x-pack/plugins/alerting_builtins/server/alert_types/index_threshold/lib/core_query_types.ts

```diff
@@ -37,17 +43,7 @@ export type CoreQueryParams = TypeOf<typeof CoreQueryParamsSchema>;
 // above.
 // Using direct type not allowed, circular reference, so body is typed to any.
 export function validateCoreQueryBody(anyParams: any): string | undefined {
-  const { aggType, aggField, groupLimit }: CoreQueryParams = anyParams;
-
-  if (aggType === 'count' && aggField) {
-    return i18n.translate('xpack.alertingBuiltins.indexThreshold.aggTypeNotEmptyErrorMessage', {
-      defaultMessage: '[aggField]: must not have a value when [aggType] is "{aggType}"',
-      values: {
-        aggType,
-      },
-    });
-  }
-
```

Comment:
I might be missing something - why do we no longer need this check? 🤔

Reply:
The UI sends parameters it doesn't need to be sending.  We could tighten this up later, but if you're using `count`, you don't need an `aggField`, we used to error if you did, now it's silently ignored.  ~shrug~